addons/fleet/models/fleet_vehicle_log_contract.py

```diff
@@ -83,6 +83,21 @@ def _compute_days_left(self):
 
     def write(self, vals):
         res = super(FleetVehicleLogContract, self).write(vals)
+        if 'start_date' in vals or 'expiration_date' in vals:
+            date_today = fields.Date.from_string(fields.Date.today())
```

Comment:
```suggestion
            date_today = fields.Date.today()
```
Is already a datetime.date